tests/OmniSharp.Tests/BufferManagerFacts.cs

```diff
@@ -10,7 +9,7 @@ namespace OmniSharp.Tests
     public class BufferManagerFacts
     {
         [Fact]
-        public async Task UpdateBufferIgnoresVoidRequests()
+        public void UpdateBufferIgnoresVoidRequests()
```

Comment:
Returning `void` here instead of `async Task` was causing XUnit to hang indefinitely when ran on mono. Did that ever get fixed?